process/support_team.md

```diff
@@ -0,0 +1,46 @@
+
+The support team doesn't have a fixed roster of staff and instead consists of a number of developers based availibility of staff and customer demand for support work. 
+
+The Support Team is run by Nick Wood, who oversees day to day operations, and manages relationships with our on-going support customers.  
+
+## Support issue lifecycle
+
+All support issues begin life as an email into support@madetech.com. These are forwarded automatically into our GrooveHQ inbox to be sorted by the Triage Team. We aim to resolve all new issues within 48 hours with one of the following resolutions:
+* Fix issue if it can be done so in < 2 hours
+* Fix issue if it is causing significant business impact to customer
+* If it is a larger piece of work, and/or non-urgent then it can be moved to the customer's backlog in Trello.
+
+Each Customer has their own Trello board for prioritising work. Issues are grouped differently for each customer but each board has the following 3 columns:
+1. 'New Issues' this is anything that's been backlogged from a Groove issue.
+2. 'Up next' issues will be automatically copied to our Support Epics board
+3. 'Recently Completed' issues are things which have been completed in the last week or so.
+
+During a weekly (bi-weekly for some customers) call with each customer Nick and the custmer will agree to a tentative commitment of work to be completed the following week by the support team. These issues will be moved to the 'Up Next' column, and copied to the Support Epics board.
+
+Every Monday everyone working on support that week will have a planning session to divide up the work for our customers across the team. Following this session Nick will follow up with each costomer to confirm what's going to be done that week.
+
+Every day support team members should attend the daily standup to report their progress ontheir support assignments, and flag up any issues or impediments which may impact the team's commitment that week.
+
+Once an issue is complete, the customer should be notified and the issue moved to the 'Done' column for the current week. 
+
+## Working in Support
+
+If you're working on support in a week (or part of a week) then you should:
+ * Attend the planning session at the start of the week, please come armed with the number of days you can commit to support that week.
+ * Attend stand-ups on days you are doing support work.
+ * Where possible we like to pair on issues, especially for our more esoteric projects, in order to spread domain knowledge throughout the company, so come willing and able to do so.
+
+### The Triage Team
+
+As part of your support week you might want to volunteer to be on the triage team. This generally consists of one of Nick, Luke, Chris, Rory plus one other person.
+
+The Triage Team rotates every week, and it's their responsibility to ensure that anything new in Groove is resolved within 48 hours (noting that moving to backlog is a suitable resolution for non-urgent issues).
```

Comment:
At least on Friday, Luke and I (as #teamTriage) were discussing whether the triage team should be clearing down this backlog once in the morning and once in the afternoon? I guess time will tell if this is workable when we have more customers using this channel, but it feels like a good customer experience to at least have a human acknowledge / run initial diagnostics within that sort of timeframe.